---
permalink: /
title: "Building a better plant - Increasing nitrogen efficiency of tropical maize"
excerpt: "About me"
author_profile: true
redirect_from: 
  - /about/
  - /about.html
---

{% include youtubePlayer.html id="ZE_Hw6npYz4" %}

Research Expertise & Core Skills
======
- Quantitative Genetics (Potato, Maize, and Wheat breeding/genomics)
- Plant Breeding for Abiotic Stress Tolerance (Heat, Drought, and Nitrogen Stress)
- Design and Analysis of Breeding Trials (single, multi-environment)
- QTL mapping, Association Studies, and Gene-based Analysis
- Population Genetics and Genomic Selection
- High-throughput Phenotyping & Statistical Modeling (Linear, Mixed, and Multivariate Models)

About me
======
Danilo is a quantitative and statistical geneticist at Rothamsted Research since 2018. He holds an M.S. and Ph.D. degree in Genetics and Plant Breeding having an extensive background in potato, maize, and wheat breeding/genomics. Danilo has a research background in classical field-based breeding activities, and experience in modern statistical genomics approaches where he focuses on understanding the genetic architecture of quantitative traits. He also received professional training at Iowa State University (Jianming Yu) and CIMMYT-Mexico (Jose Crossa). Briefly, his expertise is statistics, quantitative genetics, population genomics, QTL mapping, association studies, genomic selection, and plant breeding for abiotic stress tolerance (e.g. heat, drought, and nitrogen stress). Previously, he looked at several genomic selection approaches (i) including multi-trait models (Lyra et al. 2017), (ii) controlling population structure (Lyra et al. 2018), and (iii) incorporating dominance and copy number variation effects (Lyra et al. 2019) in maize hybrids.
Currently, he is working on four main projects with extensive collaborations (e.g. JIC, Earlham) in hexaploid wheat: 1) combining high-throughput plant phenotyping and molecular genetics for trait dissection (Lyra et al. 2020); 2) gene-based mapping of trehalose biosynthetic pathway genes for source- and sink-related traits (Lyra et al. 2020); 3) development of a quantitative genetics pipeline for the wheat Cadenza TILLING population (Lyra et al. under development), and 4) integration of genomic and metagenomic data (e.g. rhizosphere microbiomes) for trait dissection. Danilo also has experience in teaching undergraduate courses as a tutor and has also taught biology in public high schools.

Bachelor’s degree
======
I have built my short career since the early days of my undergraduate course working with plant genetic resources and pre-breeding of medicinal plants in the Brazilian semi-arid region. I also worked with on-farm conservation and participatory plant breeding (PPB). My key achievement was publishing papers as author and in collaboration with colleagues (Lyra et al. 2011a; 2011b; 2012; 2014; Brasileiro et al. 2013; Castro et al. 2011). Briefly, we were pioneers in studying the phenology and reproductive biology (pollen and seed analysis) of endemic species, as well as the local agrobiodiversity.

Master’s degree
======
In two years of training, my research goal was to design new strategies (i.e. conventional methods) of potato breeding for heat stress aiming at developing cultivars adapted to the Brazilian growing conditions (Lyra et al. 2015; Figueiredo et al. 2015). Also, I was engaged in many other projects, such as the marker-assisted selection for disease resistance (e.g. X, Y, and Leafroll viruses) (Carneiro et al. 2017; Guedes et al. 2016). Briefly, my main training activities were to conduct field testing (i.e. multi-environment trials), select and cross genotypes in cycles of recurrent selection, characterize the germplasm bank, and evaluate disease resistance in the glasshouse and field.

Doctoral degree
======
Advances in genomics are providing breeders with new tools that will allow a great leap forward in plant breeding. Thus, I extensively applied in my Ph.D. a methodology called genomic selection, which uses all molecular markers to predict the breeding values of the selection candidates, increasing genetics gains and reducing selection time (Crossa et al. 2017). Thus, my research goals were to apply genomic prediction (i) including multi-trait models (Lyra et al. 2017), (ii) controlling population structure (Lyra et al. 2018), and (iii) incorporating dominance and copy number variation effects (Lyra et al. 2019) in maize hybrids evaluated under contrasting nitrogen conditions. My published studies showed that applying multiple-trait predictions using selection indices (i.e. consider the performance under optimal and stress conditions) is a promising strategy (see educational video, link). Moreover, factors such as dominance, structural variants, and population structure can influence the accuracy of estimates of genomic breeding values.
Over my Ph.D. I received professional training in Jianming Yu’s (Iowa State University, United States) and Jose Crossa’s (CIMMYT, Mexico) research groups. In the first collaboration, I was engaged in evaluating sorghum and maize field trials while developing approaches for controlling population structure in prediction (Lyra et al. 2018). In the second collaboration, I received a short training at the Biometrics Group, studying the effects of copy variants in the phenotypic variation of complex traits in maize (Lyra et al. 2019).
I also collaborated with several colleagues in the Allogamous Plant Breeding Laboratory (link). We studied the impact of missing phenotypic data (Galli et al. 2018), marker selection (Sousa et al. 2019), Bayesian approaches (Alves et al. 2019), and uses of public databases (Moraes et al. 2020) on genomic prediction in maize. In addition, I contributed in analysing the association mapping (GWAS) data for traits related to nitrogen use efficiency (Morosini et al. 2017), and the interaction between Azospirillum brasilense and tropical maize genotypes (Vidotti et al. 2019).

Postdoctoral research
======
As a quantitative geneticist at Rothamsted Research, I have been engaged in many projects as leading author/investigator. Briefly, I have four ongoing projects, including (i) developing robust statistical and prediction strategies using the data from the Field Scanalyzer phenotyping platform (in collaboration with Malcolm Hawkesford) (Lyra et al. 2020), (ii) implementing gene-based models using exome-capture data of trehalose biosynthetic pathway genes (in collaboration with Matthew Paul), (iii) integrating genomic and metagenomic data for trait dissection (FAPESP-BBSRC Pump-Priming grant - in collaboration with Tim Mauchline), and iv) developing a quantitative genetics pipeline for the wheat Cadenza TILLING population (in collaboration with Andy Phillips and Cristobal Uauy-JIC).

<img src="https://raw.githubusercontent.com/DaniloLyra/danilolyra.github.io/master/images/editing-talk.png" alt="hi" class="inline"/>
